### PR TITLE
Issue 22971

### PR DESCRIPTION
**Pull request by bhatti-dhara**
### Fixed Issues (if relevant)

1. magento/magento2#22971: Report search terms csv export

### Manual testing scenarios (*)

1. Manual testing done by export report in Reports > Search Terms.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=23506) <br/>All committers have signed the CLA.

**Comment by Stepa4man** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @Stepa4man. Thank you for your request. I'm working on Magento instance for you

**Comment by Stepa4man** [comment]
@magento give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @Stepa4man. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @Stepa4man, here is your [new Magento instance](https://pr-23506.instances.magento-community.engineering).
Admin access: [https://pr-23506.instances.magento-community.engineering/admin](https://pr-23506.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

File: app/code/Magento/Backend/Block/Widget/Grid/Column.php [hunk 1]
```diff
@@ -337,6 +337,9 @@ public function getRowFieldExport(\Magento\Framework\DataObject $row)
             $renderedValue = call_user_func($frameCallback, $renderedValue, $row, $this, true);
         }
 
+        $renderedValue = str_replace("\r\n", " > ", trim($renderedValue));
```
Comment (kalpmehta) [inline app/code/Magento/Backend/Block/Widget/Grid/Column.php:340]: @bhatti-dhara I don't think this is the right approach to "fix" this issue. Actually, I don't see this PR addressing any valid issue.

**Comment by kalpmehta** [comment]
@bhatti-dhara I am closing this PR, I don't see any valid issue getting addressed here. Please feel free to re-open if you think we should re-consider this PR.

**Pull request closed without merging**